downstream/modules/platform/con-edge-manager-usr-dir.adoc

```diff
@@ -0,0 +1,13 @@
+[id="edge-manager-usr-dir"]
+
+= Configuration in the `/usr` directory
+
+Place configuration files in the `/usr` directory if the configuration is static and the application or service supports that configuration.
+By placing the configuration in the `/usr` directory, the configuration remains read-only and fully defined by the image.
+
+It is not feasible or desirable to place the configuration in the `/usr` directory in the following cases:
```

Comment:
Suggest changing this to be more direct: Do not place the configuration in the  `/usr` directory in the following cases: